### DISCUSSION TITLE
What's the difference between `Mouse` and `Pointer`?

### DISCUSSION THREAD

**Discussion by Magnogen** [TodePond/Habitat#93] [Q&A] [answered]
They both tell you if the mouse is down, and also where it is.

**Comment by TodePond** [comment, marked as answer]
https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent

vs

https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events

**Reply by Magnogen** [reply]
Ahhhh, okay. That makes more sense. Thank you!
